packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -1003,6 +1006,8 @@ def _check_required_class_attributes(
                 f"'synchronized_data_class' not set on {abstract_round_cls}"
             )
         if not hasattr(abstract_round_cls, "allowed_tx_type"):
+            if hasattr(abstract_round_cls, "payload"):
+                return
```

Comment:
also intermediate state of change; needed in order to introduce the new design with direct reference to `payload`